README.md

```diff
@@ -0,0 +1,69 @@
+# Using cra-template-truemark
+
+## How can we install cra-template-truemark? - Step by Step Guide
+
+1. Clone using the following repo link
+
+   #### `git clone https://github.com/TruemarkDev/cra-template-truemark.git` for https.
+
+   #### `git clone git@github.com:TruemarkDev/cra-template-truemark.git` for ssh.
+
+2. Go to the project folder
+
+   #### `cd cra-template-truemark`
+
+## How can we run the project?
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+ Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+ You may also see any lint errors in the console.
+
+## Major Dependencies and Versions
+
+- **React:** ^18.2.0
+- **React Router DOM:** ^6.15.0
+- **axios:** ^1.5.0
+- **bootstrap:** ^5.3.1
+- **i18next:** ^23.4.6
+- **react-dom:** ^18.2.0
+- **react-i18next:** ^13.2.1
+- **react-scripts:** ^5.0.1
+
+## Major Development Dependencies and Versions
+
+- **@babel/core:** ^7.22.11
+- **eslint:** ^8.48.0
+- **eslint-plugin-prettier:** ^3.4.1
+- **husky:** ^8.0.3
+- **lint-staged:** ^14.0.1
+- **prettier:** ^3.0.3
+- **sass:** ^1.66.1
+- **stylelint:** ^14.16.1
```

Comment:
We have added configuration for cypress and jest but I couldn't see the library list here which will be added when installing the template.